tests/teamcity/install-npm-deps.js

```diff
@@ -0,0 +1,54 @@
+
+/**
+ * Install local versions of dependencies specified
+ * in the project's package.json file.
+ *
+ * Usage:
```

Comment:
Hmm.. An even simpler solution might be to just copy package.json locally, install the dependencies, then remove the local copy afterwards.